BUILD.bazel

```diff
@@ -0,0 +1 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
```

Comment:
EOF newline. 

Also, we write TODOs in the form of ``TODO(#1234): Complete some action because X.``

Reply:
Done!

Reply:
Okay!